src/js/statements.rs

```diff
@@ -0,0 +1,145 @@
+use super::*;
+use crate::{
+    ast::*, error::Error::UnsupportedFeature, line_numbers::LineNumbers, pretty::*, type_::Type,
+    Result, Target,
+};
+use itertools::Itertools;
+use std::sync::Arc;
+
+fn unsupported<M: ToString, T>(label: M) -> Result<T> {
+    Err(UnsupportedFeature {
+        target: Target::JavaScript,
+        feature: label.to_string(),
+    })
+}
+
+pub fn statement<'a>(
+    _current_module: &'a [String],
```

Comment:
Some of these unused variables are here because the function signatures match the erl.rs version and I think they get used in imminent PR's